pkg/cloud/cloud.go

```diff
@@ -149,6 +152,7 @@ func (c *cloud) CreateFileSystem(ctx context.Context, volumeName string, fileSys
 	}
 
 	lustreConfiguration := &types.CreateFileSystemLustreConfiguration{}
+	metadataConfiguration := &types.CreateFileSystemLustreMetadataConfiguration{}
```

Comment:
nit: seems like this could be defined in the
```
if fileSystemOptions.MetadataConfigurationMode != "" {
...
}
```
section since it's only used in this instance